### DISCUSSION TITLE
Feature Request: Argument Text Objects

### DISCUSSION THREAD

**Discussion by kevintraver** [chrisgrieser/nvim-various-textobjs#39] [Ideas]
It would be awesome to have argument text objects, like in [targets.vim](https://github.com/wellle/targets.vim).

**Comment by chrisgrieser** [comment]
Those are already available via treesitter-textobjects 🙂

**Reply by kevintraver** [reply]
Ah, yes I see that now

Looks like I can use `@parameter.inner` and `@parameter.outer`

Thanks!
